src/layout/Tree/JsonTree/JsonTree.tsx

```diff
@@ -15,7 +15,7 @@ export interface JsonTreeProps {
   showAll?: boolean;
 
   /**
-   * The limit for the number of nodes to show when `showAll` is true.
+   * The limit for the number of nodes to show when `showAll` is false.
    */
   showAllLimit?: number;
```

Comment:
`showAllLimit` is kind of a confusing name imo, but I get you're not tryna break the API. Not even sure what a good name would be anyway, maybe `expandCount` - similar to the existing `expandDepth`

Reply:
or `expandLimit`

Reply:
eh none of those are great haha

Reply:
Agreed odd name but she just followed the other name. I think fine for now.

Reply:
yea didn't want to change it as it'd be a breaking change